### ISSUE TITLE
uniblock demo can be molested by output

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/notcurses#183]
Now that we're focusing on small details, the uniblock demo still shows some deformities on complex codeblocks. It's easy enough to see. Can we not lock this down once and for all?

**Comment by dankamongmen** [comment]
This seems to be fairly terminal-dependent, possibly font dependent, fun fun.

**Comment by dankamongmen** [comment]
In kitty, we're broken on the 6th line from block 1 on. The offending character appears to be \u00ad, "SOFT HYPHEN":

```
[schwarzgerat](0) $ unicode 00ad
U+00AD SOFT HYPHEN
UTF-8: c2 ad UTF-16BE: 00ad Decimal: &#173; Octal: \0255
­
Category: Cf (Other, Format); East Asian width: A (ambiguous)
Unicode block: 0080..00FF; Latin-1 Supplement
Bidi: BN (Boundary Neutral)


[schwarzgerat](0) $ 
```

without this character, we do fine. With it, kitty is messed up, and remains so *even if we insert an `ncplane_erase`*.

**Comment by dankamongmen** [comment]
This seems fixed by the work for #259. In alacritty, VTE, and kitty we no longer see this breakage, or at least it's not persistent. The bit about `ncplane_erase()` that blew my mind above is explained in #259. Excellent!
